apps/backend/config/app_config.ts

```diff
@@ -51,9 +51,54 @@ export default class AppConfig {
   }
 
   getSSLConfig() {
+    let sslKey, sslCert, sslCA;
+
+    if (typeof this.get('DATABASE_SSL_KEY') === 'string') {
+      if (this.get('DATABASE_SSL_KEY')?.indexOf('-BEGIN') !== -1) {
+        sslKey = this.get('DATABASE_SSL_KEY')
+      } else {
+        // Verify file exists
+        if (fs.statSync(this.get('DATABASE_SSL_KEY')!).isFile()) {
+          sslKey = fs.readFileSync(this.get('DATABASE_SSL_KEY')!);
+        } else {
+          throw new Error('SSL Key file does not exist');
+        }
+      }
+    }
+
+    if (typeof this.get('DATABASE_SSL_CERT') === 'string') {
+      if (this.get('DATABASE_SSL_CERT')?.indexOf('-BEGIN') !== -1) {
+        sslCert = this.get('DATABASE_SSL_CERT')
+      } else {
+        // Verify file exists
+        if (fs.statSync(this.get('DATABASE_SSL_CERT')!).isFile()) {
+          sslCert = fs.readFileSync(this.get('DATABASE_SSL_CERT')!);
+        } else {
+          throw new Error('SSL Cert file does not exist');
+        }
+      }
+    }
+
+    if (typeof this.get('DATABASE_SSL_CA') === 'string') {
+      if (this.get('DATABASE_SSL_CA')?.indexOf('-BEGIN') !== -1) {
+        sslCA = this.get('DATABASE_SSL_CA')
+      } else {
+        // Verify file exists
+        if (fs.statSync(this.get('DATABASE_SSL_CA')!).isFile()) {
+          sslCA = fs.readFileSync(this.get('DATABASE_SSL_CA')!);
+        } else {
+          throw new Error('SSL CA file does not exist');
+        }
+      }
+    }
+
+
     return Boolean(this.get('DATABASE_SSL'))
```

Comment:
dotenv parses all variables as strings.  the Boolean function treats any string aside from '' as truthy and will therefore return true even if the string is 'false'.  I think this ternary operator will always follow the path of returning the object unless the variable is left empty basically.